src/ValueObject/StaticNonPhpFileSuffixes.php

```diff
@@ -11,13 +11,4 @@ final class StaticNonPhpFileSuffixes
      */
     public const SUFFIXES = ['neon', 'yaml', 'xml', 'yml', 'twig', 'latte', 'blade.php', 'tpl'];
```

Comment:
I wonder how is this class needed now. What prevents it from being removed?

Reply:
it seems you are fine with dropping this constant and you don't mind BC.

therefore I moved the constant to the last remainig consumer and removed the whole class